components/buttons/button.css

```diff
@@ -113,5 +120,10 @@ color:var(--complement-btn-text-color);
     background-color:var(--primary-color);
     opacity:0.7;
    }
-  
+  /************************/
+.link-btn{
+    text-decoration:none;
+    border-radius: .5rem;
```

Comment:
Please go constant about units. Use the same units if possible.
For example here: The unit used is rem and in line 105, it is in %. Please correct me if I said something wrong.

Reply:
will see to it !

Reply:
Okay great. Everything else is good.